src/parser/vast_parser.js

```diff
@@ -218,10 +218,10 @@ export class VASTParser extends EventEmitter {
   /**
    * Builds a VASTResponse which can be returned.
    * @param  {Array} ads - An Array of unwrapped ads
-   * @return {VASTResponse}
+   * @return {createVASTResponse}
```

Comment:
same here with the type. also since `createVASTReponse` only has 3 properties, you could pass them in the function. ex:
```
const response = createVASTResponse({
    ads,
    errorURLTemplates: this.getErrorURLTemplates(),
    version: this.vastVersion,
})
```